tests/kernel_bundle/use_kernel_bundle_verify_that_kernel_was_invoked.cpp

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This test obtain kernel bundle in bundle_state::input, then for this kernel
+//  calls set_spec_const with some different value, then call sycl::build to
+//  have kernel bundle in bundle_state::executable state and calls
+//  sycl::handler::use_kernel_bundle for this bundle, then get spec const value.
+//
+//  The test verifies that the pec const value is equal to expected velue.
```

Comment:
```suggestion
//  The test verifies that the specialization constant value is equal to expected value.
```

Reply:
I've applied this comment [here](https://github.com/KhronosGroup/SYCL-CTS/pull/225/commits/e157b8bd72174f5b18b4e387b9dfb4e353573bd7#diff-d950e7de8902d28e945f2971a8e619986ed79fcf9076a4205047beac0da4175aL10-L11)